docs/content.zh/docs/dev/table/sql/show.md

```diff
@@ -507,10 +507,21 @@ SHOW CURRENT CATALOG
 ## SHOW DATABASES
 
 ```sql
-SHOW DATABASES
+SHOW DATABASES [ ( FROM | IN ) catalog_name] [ [NOT] (LIKE | ILIKE) <sql_like_pattern> ]
```

Comment:
Out of curiosity. Do you know chinese?

Reply:
Not really :) Used some translating apps + checked existing chinese documentation, so that this feature is documented somehow.

Reply:
Let's keep it, but I'd prefer not to do that. I don't speak chinese and I don't feel comfortable committing something I don't understand. There is no way for me to say if you did not write something offending there. I believe you did not, but I can not tell for sure.

Reply:
Agree. Removed